meetuppizza/tests.py

```diff
@@ -8,3 +25,50 @@ def test_landing_page_is_there(self):
   def test_page_contains_pizza(self):
     response = self.client.get('/')
     self.assertContains(response, "pizza")
+
+  def test_signup_redirects(self):
+    response = self.client.post('/sign_up', self.params, follow=True)
+    self.assertRedirects(response, '/')
+
+  def test_user_is_created(self):
```

Comment:
Make sure the names of your test methods include all the relevant what/when/where/why/how of the tests they describe. When is this user created?